guides/image-sources/provisioning-installer-bootdisk-subnet.plantuml

```diff
@@ -0,0 +1,60 @@
+@startuml
+
+!include foreman.pstyle
+!$networkboot = 1
+
+title Installer-based provisioning with subnet bootdisk
+
+actor       User
+participant "Provisioned\nHost" as Host
+participant "Foreman" as Foreman
+participant "Foreman\nProxy" as Proxy
+'participant "Pulp\n(Katello)" as Pulp
+'participant TFTP
+participant DHCP
+participant DNS
+!if ($puppet)
+participant "Puppet\nserver" as Puppet
+!endif
+
+note over Host : powered off
+
+User -> Foreman : downloads the bootdisk of the subnet
+note over User : writes the bootdisk\nto a USB/CD/DVD drive
```

Comment:
You might want to change `USB/CD/DVD drive` to "external storage device". I believe that is the generic term.